src/scripts/clipboard.js

```diff
@@ -0,0 +1,59 @@
+import browser from './browser';
+
+/**
+ * Copies text to the clipboard using the textarea.
+ * @param {string} text - Text to copy.
+ * @returns {Promise<void>} Promise.
+ */
+function textAreaCopy(text) {
+    const textArea = document.createElement('textarea');
+    textArea.value = text;
+    document.body.appendChild(textArea);
+    textArea.focus();
+
+    if (browser.iOS) {
+        // https://stackoverflow.com/a/46858939
```

Comment:
This workaround wasn't needed for me on iOS 15... I'm not sure if we should limit it to affected versions or not. What do you think?

Reply:
iOS 13.4 should be fine with `writeText` (https://stackoverflow.com/a/61868028)
I've added a version check for 10.x

Reply:
Sounds like we should test for <13.4. I believe there are some devices that have a max iOS version of 12.

Reply:
Done.
On the other hand, iOS 13.4+ won't get that check - it supports `Clipboard.writeText`.